pylintrc

```diff
@@ -46,7 +46,9 @@ disable=line-too-long,
         no-member,
         import-error,
         fixme,
-        locally-disabled
+        locally-disabled,
+        len-as-condition, # we don't really honor this one
+        syntax-error # happens when pylint tries to process a build in place .so extension
```

Comment:
I couldn't find this in the docs. Do you have a reference? I just wanted to take a look out of curiosity.